src/test/java/easytests/support/TopicsSupport.java

```diff
@@ -166,7 +166,8 @@ public void assertModelsListEquals(List<TopicModelInterface> expected, List<Topi
 
         Integer i = 0;
         for(TopicModelInterface topicModel: expected) {
-            this.assertEquals(topicModel, actual.get(i++));
+            this.assertEquals(topicModel, actual.get(i));
+            i++;
```

Comment:
@malinink 
You know, the elegance and beauty of increment operator is lost when we increment it like this :)

Reply:
@Ielay In my opinion code with `i++` have a potential failure on adding one more line that uses also `i` in that for loop. Its just experience